rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/CommandStorageQueryFactory.java

```diff
@@ -26,17 +26,20 @@
 import org.spine3.base.Error;
 import org.spine3.base.Failure;
 import org.spine3.server.storage.CommandStorageRecord;
+import org.spine3.server.storage.jdbc.query.Query;
 import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
 import org.spine3.server.storage.jdbc.util.IdColumn;
 
+import java.util.Iterator;
+
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.TABLE_NAME;
 
 /**
  * This class creates queries for interaction with {@link CommandTable}.
  *
  * @author Andrey Lavrov
  */
-@SuppressWarnings("TypeMayBeWeakened")
+@SuppressWarnings({"TypeMayBeWeakened", "OverlyCoupledClass"})
```

Comment:
Pls add a comment that we don't split this class because it has methods of the same type.